lib/line_item.go

```diff
@@ -212,6 +214,7 @@ type UpdateLineItemCriteria struct {
 	SurveyURLParams     []*URLParameter    `json:"surveyURLParams" valid:"optional"`
 	SurveyTestURLParams []*URLParameter    `json:"surveyTestURLParams" valid:"optional"`
 	Sources             *[]*LineItemSource `json:"sources,omitempty" valid:"optional"`
+	SurveyTestingNotes  *string           `json:"surveyTestingNotes,omitempty" valid:"optional"`
```

Comment:
Can you add test coverage for these changes to `line_item.go`?

Reply:
Ok.

Reply:
How do I test?